views/tables/ParticipantsTable/index.jsx

```diff
@@ -63,25 +63,46 @@ const ParticipantsTable = (props) => {
   ]
   const cellRenderer = (participant, property) => {
     switch (property) {
+      case 'days':
+        if (participant?.synced && participant?.Consent) {
+          const lastSyncedDate = dayjs(participant.synced)
+          const consentDate = dayjs(participant.Consent)
+
+          return `${lastSyncedDate.diff(consentDate, 'd')} days`
+        } else if (
+          !participant?.synced &&
+          participant?.Consent &&
+          dayjs(participant.Consent).isValid()
+        ) {
+          const consentDate = dayjs(participant.Consent)
+
+          return `${dayjs().diff(consentDate, 'd')} days`
+        }
```

Comment:
I _think_ this logic can be cleaned up a bit. How's this look to you?

```suggestion
        const participantConsent = dayjs(participant.Consent)
        const consentDate = participant.Consent && participantConsent.isValid() ? participantConsent : null
        const lastSyncedDate = participant.synced ? dayjs(participant.synced) : dayjs()
        
        return consentDate ? `${lastSyncedDate.diff(consentDate, 'd')} days` : ''
```

Reply:
[0af550e](https://github.com/AMP-SCZ/dpdash/pull/600/commits/0af550ebd947fb7c731cba9479f9382fa5ea9138)